### PR TITLE
build: TypeScript

### PR DESCRIPTION
**Pull request by tpluscode**
migrates types from @types/rdf-loaders-registry

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,6 @@ coverage
 node_modules
 .DS_Store
 coverage/
+*.js
+!test/*.js
```
Comment (ludovicm67) [inline .gitignore:7]: After this change, is there still a risk to get unwanted `.js` files?

Comment (ludovicm67) [reply .gitignore:7]: And without those two lines in the `.gitignore`, is there a risk to get unwanted JS files?

Comment (tpluscode) [reply .gitignore:7]: Well yes, if you `tsc` locally

Comment (ludovicm67) [reply .gitignore:7]: Ok, so LGTM! 👍

**Pull request merged**